build-tools/mono-runtimes/mono-runtimes.props

```diff
@@ -27,6 +27,12 @@
     <!-- LLVM -->
     <_LlvmNeeded Condition=" '$(AndroidSupportedTargetAotAbis)' != '' ">yes</_LlvmNeeded>
     <_LlvmCanBuild64 Condition=" '$(HostBits)' == '64' ">yes</_LlvmCanBuild64>
+    <_LlvmNeededWindows32></_LlvmNeededWindows32>
```

Comment:
I don't understand why this line and the following two lines are required. If the `@Condition` fails, they'll default to the empty string/undefined anyway.